static/js/click_handlers.js

```diff
@@ -649,18 +657,34 @@ exports.initialize = function () {
                 }
             }
         });
-
+        function group_or_stream(temp, selector, flag) {
```

Comment:
I would suggest to make this function more easy and clear to understand with few changes like
- Rename `temp` to more precise name
- instead of passing `flag` argument 0 or 1, more precise is better, suppose `editable` or `checkmark` etc.
- removing second `if (!group_id)` condition as it is obvious

If we want to add new editable field, it should be easy to just add `if..else` logic in this function and add elements in `map`.

Reply:
* how about `event` in place of `temp` ?
* ohk so, I should pass `editable` or `checkmark` strings in place of 0 or 1 ( i.e. get rid of list)
* actually sometimes linting asks to write an `else if` instead of only `else` so I got a habit to write it like that, removed it.